ddtrace/tracer/telemetry_test.go

```diff
@@ -67,26 +68,32 @@ func TestTelemetryEnabled(t *testing.T) {
 	})
 
 	t.Run("telemetry customer or dynamic rules", func(t *testing.T) {
-		rule := TagsResourceRule(map[string]string{"tag-a": "tv-a??"},
-			"resource-*", "op-name", "test-serv", 0.1)
+		rules := []SamplingRule{
+			TagsResourceRule(map[string]string{"tag-a": "tv-a??"}, "resource-*", "op-name", "test-serv", 0.1),
+			SpanNameServiceRule("op-name", "test-serv", 0.1),
+		}
```

Comment:
it looks like you want to test span rules here. This test originally was trying to test remote rules of `Customer` or `Dynamic` provenance. Note the skip when `if prov == Local `.

In the current implementation, there will only be span rules in exactly the opposite situation - meaning only `Local` rules, but not the other two types.

I recognize that current rules and telemetry impl will support the change in this PR. However, in order to convey the right information for future readers, can we move the span rule test to its own test case and also add a comment that says there are only `Local` span rules as of now?

Reply:
@yuanyuanzhao3 Done.